lib/docker.ts

```diff
@@ -23,15 +24,61 @@
 import Docker from 'dockerode'
 import waitOn from 'wait-on'
 
+import { type Stream, PassThrough } from 'stream'
+import { join, resolve, sep } from 'path'
+import { existsSync, readFileSync } from 'fs'
+import { XMLParser } from 'fast-xml-parser'
+
 export const docker = new Docker()
 
 const CONTAINER_NAME = 'nextcloud-cypress-tests'
 const SERVER_IMAGE = 'ghcr.io/nextcloud/continuous-integration-shallow-server'
 
+const VENDOR_APPS = {
+	text: 'https://github.com/nextcloud/text.git',
+	viewer: 'https://github.com/nextcloud/viewer.git',
+	notifications: 'https://github.com/nextcloud/notifications.git',
+	activity: 'https://github.com/nextcloud/activity.git',
+}
```

Comment:
Fancy!! :+1:

Reply:
We need this otherwise there are failures because at least viewer is set to be "always enabled" meaning if you use the shallow server you will get errors because viewer is missing :(

Reply:
I wasn't sarcastic! It's a good idea :grin: